stable/dynamic-gateway-service/templates/statefulset.yaml

```diff
@@ -289,6 +289,28 @@ spec:
               value: {{ .Values.datapower.env.datapowerLogLevel | quote }}
             - name: "PEERING_LOG_LEVEL"
               value: {{ .Values.datapower.env.peeringLogLevel }}
+            - name: "GWD_POLICY_DISABLE_LOG"
+{{- if eq .Values.datapower.policyDisableLog "true" }}
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}
+            - name: "GWD_POLICY_DISABLE_XSLT"
+{{- if eq .Values.datapower.policyDisableXSLT "true" }}
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}  - name: "GWD_POLICY_DISABLE_GATEWAYSCRIPT"
```

Comment:
If this doesn't result in a renderer error, it's still a poor formatting practice. `-name` should be on its own line.

Reply:
good catch, fixed